docs/general/run-a-node/prerequisites/hardware-recommendations.md

```diff
@@ -1,218 +1,197 @@
-storage checkpoints with BadgerDB, periodic BadgerDB compactions...
-
-If the system will not have enough memory, that will result in the Oasis node
-process being killed forcefully by the OOM process.
-Oasis node being killed forcefully could lead to BadgerDB state corruption
-and/or losing stake.
-
-:::
-
-### Storage
-
-* Minimum: 300+ GB of SSD or NVMe fast storage
-* Recommended: 500+ GB of SSD or NVMe fast storage
-
-:::caution
-
-Consensus state is stored in an embedded [BadgerDB] database which was
-[designed to run on SSDs][badgerdb-ssds].
-
-Hence, we **strongly discourage** trying to run a node that stores data
-**on classical HDDs**.
-
-:::
 
-:::info
+### Memory
 
-The network accumulates state over time. The speed at which the state grows
-depends on the network's usage.
+* Consensus validator or non-validator node:
+  * Minimum: 4 GB of ECC RAM
+  * Recommended: 8 GB of ECC RAM
 
-For example, the Mainnet accumulated over 250 GB of state in ~11 months between
-the [Cobalt upgrade] (Apr 28, 2021) and Mar 15, 2022.
+* All ParaTime compute or client nodes:
+  * Minimum: 8 GB of ECC RAM
+  * Recommended: 16 GB of ECC RAM
 
-:::
+* Emerald Web3 gateway with PostgreSQL:
+  * Minimum: 4 GB of ECC RAM
+  * Recommended: 8 GB of ECC RAM
 
 :::info
 
-Node doesn't need to keep the state from before the latest dump & restore
-upgrade (e.g. before the [Cobalt upgrade]). Historical state can be archived
-separately.
+During regular workload your node will operate with less than the minimum amount
+of memory. However, at certain time points, it will absolutely require more
+memory. Examples of such more resource intensive time points are the initial
+state sync, BadgerDB migration when upgrading a node to a new major version of
+the Oasis Core, generating storage checkpoints with BadgerDB, periodic BadgerDB
+compactions...
 
 :::
 
-:::info
-
-It is also possible to configure the Node to _not_ keep all the state from the
-genesis onward, reducing the amount of storage needed to keep the network's
-state.
 
-To do that, set the **`consensus.tendermint.abci.prune.strategy`** and
-**`consensus.tendermint.abci.prune.num_kept`** parameters appropriately in your
-[node's configuration].
-
-:::
-
-[BadgerDB]: https://dgraph.io/docs/badger/
-[badgerdb-ssds]: https://dgraph.io/docs/badger/design/
-[Cobalt upgrade]: ../../mainnet/previous-upgrades/cobalt-upgrade.md
-[node's configuration]: ../set-up-your-node/run-validator.md#configuring-the-oasis-node
+### Storage
 
-## ParaTime Nodes
+* Consensus validator or non-validator node:
+  * Minimum: 400+ GB of SSD or NVMe fast storage
+  * Recommended: 700+ GB of SSD or NVMe fast storage
 
-To run a ParaTime node, your system should meet the following requirements:
+* Emerald ParaTime compute or client node:
+  * Add these to the Consensus Layer storage requirements
+  * Minimum: 400+ GB of SSD or NVMe fast storage
+  * Recommended: 700+ GB of SSD or NVMe fast storage
```

Comment:
@matevz Which suggestion sounds better?

```suggestion
  * In addition to the storage requirements for consensus state, the Emerald ParaTime requires additional storage
  * Minimum: 400+ GB of SSD or NVMe fast storage
  * Recommended: 700+ GB of SSD or NVMe fast storage
```

```suggestion
  * In addition to the storage requirements for consensus state, it requires at least additional: 400+ GB of SSD or NVMe fast storage
  * In addition to the storage requirements for consensus state, we recommend additional: 700+ GB of SSD or NVMe fast storage
```

Reply:
Maybe replace the initial bullet with

```suggestion
* Emerald ParaTime compute or client node (in addition to the consensus storage requirements):
  * Minimum: 400+ GB of SSD or NVMe fast storage
  * Recommended: 700+ GB of SSD or NVMe fast storage
```

And the same for Cipher below. Maybe move the Cipher next to Emerald so all ParaTimes are grouped together?